src/extensions/mountainview/MountainView/MountainView.tsx

```diff
@@ -43,14 +51,18 @@ export const openViewsReducer: React.Reducer<View[], OpenViewsAction> = (state:
                 }
             }
         }
-        lastViewId ++
-        const v = new View(action.pluginType, plugin, lastViewId)
+        lastViewIdNum ++
+        const v = new View(action.pluginType, plugin, lastViewIdNum + '')
         v.activate = true // signal to set this as active
-        return [...state, v]
+        v.area = action.area
+        return [...state, v].sort((a, b) => (a.plugin.label > b.plugin.label ? 1 : a.plugin.label < b.plugin.label ? -1 : 0))
```

Comment:
We always sort the tabs alphabetically by label. I think this is a reasonable way to keep things organized. Then we also don't need to worry about providing tab-move functionality (except between the north/south areas)